packages/experiments/src/components/fluent/theme/FluentColors.ts

```diff
@@ -1,5 +1,17 @@
 /* tslint:disable:no-any */
-export const GrayColors: any = {
+export const CommunicationColors: any = {
+  communicationShade30: '#004578',
+  communicationShade20: '#005a9e',
+  communicationShade10: '#106ebe',
+  communicationPrimary: '#0078d4',
+  communicationTint10: '#2b88d8',
+  communicationTint20: '#71afe5',
+  communicationTint30: '#c7e0f4',
+  communicationTint40: '#deecf9',
+  communicationTint50: '#eff6fc'
+};
+
+export const NeutralColors: any = {
```

Comment:
It's confusing calling this the neutrals, but then everything inside is called gray.